common/version.h

```diff
@@ -34,7 +34,7 @@
  * Manifest: https://github.com/EQEmu/Server/blob/master/utils/sql/db_update_manifest.txt
  */
 
-#define CURRENT_BINARY_DATABASE_VERSION 9185
+#define CURRENT_BINARY_DATABASE_VERSION 9186
```

Comment:
Suggest 9187 due to https://github.com/EQEmu/Server/pull/2297

Reply:
9187 used by https://github.com/EQEmu/Server/pull/2303/files#diff-a4af6a2b9014b5d7d6093baf53e35cce68e6a4c2bd28d21cfd2b17e1ab8036e4R37

Reply:
Well, that too!